### DISCUSSION TITLE
[p.287] react-router-dom 버전 문제

### DISCUSSION THREAD

**Discussion by sijune** [fsoftwareengineer/todo-application#51] [질문과 답변]
npm install react-router-dom 설치 시 버전은 6으로 설치됩니다. 따라서 Switch가 작동을 안 할 수 있는데, 해결책은 2가지로 아래와 같습니다.

1. 버전 5로 설치
$ npm install react-router-dom@5

2. Switch대신 Routes 사용
![image](https://user-images.githubusercontent.com/69071211/148667662-ac899c10-2d4e-4410-9760-f8881fdd8703.png)

참고부탁드립니다.

**Comment by leeyou34** [comment]
오우! 저도 2번 방법으로 하니깐, 하단에 copyright이 떴어요! =D 완전 감사합니당!!!

**Comment by KIMHAUN** [comment]
왕 대박 감사합니다  ♥ 저같은 왕초보를 위해 첨언하자면 맨 위에 import하는 거도 수정해야합니다 ~
